pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -434,6 +435,7 @@ func (b *HybridBotanist) DeployKubeControllerManager() error {
 // cloud-controller-manager deployment.
 func (b *HybridBotanist) DeployCloudControllerManager() error {
 	defaultValues := map[string]interface{}{
+		"enabled":           true,
```

Comment:
Now that I see this: Why is there the `enabled` value in the cloud-controller-manager and alicloud-cloud-controller-manager charts at all? It's always set to `true` here and the differentiation which chart is rendered is also provided by the cloud-specific botanist (as it returns the chart name).
Hence, let's remove the `enabled` value completely?!

Reply:
You are right. Control plane not like the shoot-core. The charts are rendered separately.